### PR TITLE
[Relay] added breaking change.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by isra-fel — COMMENTED** [review]
Breaking changes should have a targeted date. Will it happen in our next breaking change relase this May?

**Comment by LucasYao93** [comment]
> Breaking changes should have a targeted date. Will it happen in our next breaking change release this May?
Hi yeming,
I want to confirm our release steps.

First, we will release module with breaking change.
Second, we will release generate module.

If the steps are correct. Our target date is May.

File: src/Relay/Relay/Cmdlets/Namespace/SetAzureRelayNamespace.cs [hunk 1]
```diff
@@ -23,6 +24,7 @@ namespace Microsoft.Azure.Commands.Relay.Commands.Namespace
     /// <summary>
     /// 'Set-AzRelayNamespace' Cmdlet updates the specified Relay Namespace
     /// </summary>
+    [GenericBreakingChange("Set-AzRelayNamespace will be removed in an upcoming breaking change release, you can use Update-AzRelayNamespace in a future release")]
```
Comment (BethanyZhou) [inline src/Relay/Relay/Cmdlets/Namespace/SetAzureRelayNamespace.cs:27]: Please add a target date or target version in breaking change message.

**Review by BethanyZhou — CHANGES_REQUESTED** [review]
The target version should be 2.0.0 for Az.Relay

File: src/Relay/Relay/Cmdlets/Namespace/SetAzureRelayNamespace.cs (hunk 1 above)
Comment (LucasYao93) [reply src/Relay/Relay/Cmdlets/Namespace/SetAzureRelayNamespace.cs]: Done

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**